lib/builders/MaterialBuilder.ts

```diff
@@ -29,12 +29,25 @@ export class MaterialBuilder {
         // Update the resolved version, in case it was `node`.
         this._options.version = versionData.version;
 
+        let deps = DependencyResolver.resolve(
+          this._getModuleEntry(versionData),
+          this._options.modules
+        );
+
+        if (this._options.excludeModules) {
+            // Remove modules that were explicitly flagged to be excluded from the build
+            deps._flat = deps._flat.filter(dep => this._options.excludeModules.indexOf(dep) === -1);
+            deps._mainModule.dependencies = deps._mainModule.dependencies.filter(
```

Comment:
This looks a bit "hacky" because the properties are prefixed with an underscore, but it's fine like that since there is a comment 👍